src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Selection.js

```diff
@@ -68,6 +82,22 @@ export default class Selection extends React.Component<Props> {
                 this.handleListSelectionChange
             );
 
+            this.changeListOptionsDisposer = reaction(
+                () => this.buildRequestOptions(requestParameters, resourceStorePropertiesToRequest, formInspector),
+                (requestOptions) => {
+                    const listStore = this.listStore;
```

Comment:
This reaction runs too often. I have added a `resource_store_properties_to_request` option to a text field, and on every single keystroke this code is ran and sends another request... This should actually be done only when a field calls its `onFinish` handler.

The `ResourceLocator` field has already a finished handler, which acts this way.